### DISCUSSION TITLE
비로그인 시 프로필 페이지 이동 처리

### DISCUSSION THREAD

**Discussion by eeseung** [prgrms-fe-devcourse/FEDC4_CUCUMIS_Pocojang#64] [General]
비로그인 상태에서 Navbar - 프로필 이동한 경우, `/profile/:id`로 이동해야 하는데 id가 없어 `/profile`로 이동하게 됩니다. 라우팅 처리와 어떻게 보여주는게 좋을지 논의가 필요합니다.

**Comment by kutta97** [comment]
저번 주 기획 회의때 얘기 나눴을 때는,
"`로그인 사용자만 이용가능한 서비스입니다`" 라는 내용의 Modal 을 띄우고, `confirm` 버튼으로 "로그인"페이지로 이동
약간 이런 느낌으로 결론이 났었는데, 회의록에 문서화가 제대로 되어있지 않았던 것 같습니다!

**Reply by eeseung** [reply]
그럼 일단 따로 처리하지 않고 그대로 두겠습니다! (이제야 답변을 남기네요.. 💦)
